### DISCUSSION TITLE
FL33 Slip-rate imposed on a DR boundary: Implementation scheme

### DISCUSSION THREAD

**Discussion by jwjeremy** [SeisSol/SeisSol#705] [General]
Hi @Thomas-Ulrich, I would like to ask for the FL33 parameter setting. Is the slip rate implemented to the gaussian points (grey point in the figure subset) or the mesh elements (black point in the figure subset)? 
CC @AliceGabriel 

<img width="770" alt="image" src="https://user-images.githubusercontent.com/44199955/197114781-6bf4ad54-226e-4393-9ff1-54700ec30452.png">
Figure C1 retrieved from Wollherr, S., Gabriel, A. A., & Uphoff, C. (2018).

Many thanks!

**Comment by Thomas-Ulrich** [comment]
It is at the Gauss points.
The code is here:
https://github.com/SeisSol/SeisSol/blob/master/src/DynamicRupture/FrictionLaws/ImposedSlipRates.h#L48
Note also that we have a new set of quadrature points (Dunavant), that can be chosen via cmake (`-DDR_QUAD_RULE=dunavant`). It has a symmetrical distribution within the element.

**Reply by jwjeremy** [reply]
I see, thank you so much!
